examples/tutorial_server_eventfilter.c

```diff
@@ -0,0 +1,282 @@
+                         size_t outputSize, UA_Variant *output) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "Creating event");
+    UA_StatusCode retval;
+    UA_NodeId* eventNodeId;
+    eventNodeId = (UA_NodeId*)
+        UA_Array_new(eventTypesCount, &UA_TYPES[UA_TYPES_NODEID]);
+    /* set up event */
+    for(size_t i = 0 ; i < eventTypesCount; i++) {
+        retval = setUpEvent(server, &eventNodeId[i], eventTypes[i], false,i);
+        if(retval != UA_STATUSCODE_GOOD) {
+            UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                           "Creating event failed. StatusCode %s", UA_StatusCode_name(retval));
+            return retval;
+        }
+    }
+    for(size_t i = 0 ; i < eventTypesCount; i++) {
+        retval = UA_Server_triggerEvent(server, eventNodeId[i],
+                                         UA_NODEID_NUMERIC(0, UA_NS0ID_SERVER),
+                                         NULL, UA_TRUE);
+        if(retval != UA_STATUSCODE_GOOD)
+            UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                           "Triggering event failed. StatusCode %s", UA_StatusCode_name(retval));
+    }
+    return retval;
+}
+
+static UA_StatusCode
+generateRandomEventMethodCallback(UA_Server *server,
+                             const UA_NodeId *sessionId, void *sessionHandle,
+                             const UA_NodeId *methodId, void *methodContext,
+                             const UA_NodeId *objectId, void *objectContext,
+                             size_t inputSize, const UA_Variant *input,
+                             size_t outputSize, UA_Variant *output) {
+    UA_LOG_INFO(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND, "Creating event");
+    size_t random = (size_t) rand() % eventTypesCount;
+    /* set up event */
+    UA_NodeId eventNodeId;
+    UA_StatusCode retval = setUpEvent(server, &eventNodeId, eventTypes[random], true,random);
+    if(retval != UA_STATUSCODE_GOOD) {
+        UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                       "Creating event failed. StatusCode %s", UA_StatusCode_name(retval));
+        return retval;
+    }
+    retval = UA_Server_triggerEvent(server, eventNodeId,
+                                    UA_NODEID_NUMERIC(0, UA_NS0ID_SERVER),
+                                    NULL, UA_TRUE);
+    if(retval != UA_STATUSCODE_GOOD)
+        UA_LOG_WARNING(UA_Log_Stdout, UA_LOGCATEGORY_USERLAND,
+                       "Triggering event failed. StatusCode %s", UA_StatusCode_name(retval));
+    return retval;
+}
+
+/**
+ * Now, all that is left to do is to create a method node which uses our callback. We do not
+ * require any input and as output we will be using the `EventId` we receive from ``triggerEvent``. The `EventId` is
+ * generated by the server internally and is a random unique ID which identifies that specific event.
+ *
+ * This method node will be added to a basic server setup.
```

Comment:
I don't get this description "This method node will be added to a basic server setup."

Reply:
We changed the description to "This method node will be added to the server."
Is that any clearer?